maat-court-data-api/src/main/java/gov/uk/courtdata/enums/FdcContributionsStatus.java

```diff
@@ -0,0 +1,8 @@
+package gov.uk.courtdata.enums;
+
+public enum FdcContributionsStatus {
+    REQUESTED,
+    SENT,
+    REPLACED,
+    WAITING_ITEMS
+}
```

Comment:
Looking at the DB I think we are missing the status of `INVALID`

Reply:
I had left out as it wasn't a use case we were looking into, but absolutely valid. Will add.